src/main/resources/org/damap/base/db/changeLog-root.yaml

```
@@ -21,3 +21,5 @@ databaseChangeLog:
       file: org/damap/base/db/changeLog-4.x/changeLog-4.0.0_2.yaml
   - include:
       file: org/damap/base/db/changeLog-4.x/changeLog-4.0.3_1.yaml
+  - include:
+      file: org/damap/base/db/changeLog-4.x/changeLog-4.0.0_4.yaml
```

```suggestion
      file: org/damap/base/db/changeLog-4.x/changeLog-4.0.3_2.yaml
```
Also requires renaming the file

---

Done